### ISSUE TITLE
body_add_gg adds three copies of the plot

### ISSUE THREAD

**Issue by mattantaliss** [davidgohel/officer#323]
When adding a plot to a document with `body_add_gg` and units not explicitly set, three copies of the plot are added with decreasing image sizes. Here's a reprex:

``` r
library(officer)
library(magrittr)
library(ggplot2)

p <- ggplot(data = mtcars, aes(x = wt, y = mpg)) +
  geom_point()

# Produces three images of decreasing size
# (assumption: body_add_img is taking all of c("in", "cm", "mm") and adding the
# same image at each of those units)
read_docx() %>%
  body_add_gg(p) %>%
  print("test_3.docx")

# Produces one image
read_docx() %>%
  body_add_gg(p, units = "in") %>%
  print("test_1.docx")
```

<sup>Created on 2020-09-02 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

Here's my sessionInfo:

> R version 4.0.2 (2020-06-22)
> Platform: x86_64-w64-mingw32/x64 (64-bit)
> Running under: Windows 10 x64 (build 19041)
> 
> Matrix products: default
> 
> locale:
> [1] LC_COLLATE=English_United States.1252  LC_CTYPE=English_United States.1252   
> [3] LC_MONETARY=English_United States.1252 LC_NUMERIC=C                          
> [5] LC_TIME=English_United States.1252    
> 
> attached base packages:
> [1] stats     graphics  grDevices utils     datasets  methods   base     
> 
> other attached packages:
> [1] ggplot2_3.3.2      magrittr_1.5       officer_0.3.13.011
> 
> loaded via a namespace (and not attached):
>  [1] rstudioapi_0.11  xml2_1.3.2       tidyselect_1.1.0 munsell_0.5.0   
>  [5] uuid_0.1-4       colorspace_1.4-1 R6_2.4.1         rlang_0.4.7     
>  [9] dplyr_1.0.1      tools_4.0.2      grid_4.0.2       gtable_0.3.0    
> [13] withr_2.2.0      ellipsis_0.3.1   digest_0.6.25    tibble_3.0.3    
> [17] lifecycle_0.2.0  crayon_1.3.4     zip_2.1.0        farver_2.0.3    
> [21] purrr_0.3.4      vctrs_0.3.2      glue_1.4.1       labeling_0.3    
> [25] compiler_4.0.2   pillar_1.4.6     generics_0.0.2   scales_1.1.1    
> [29] pkgconfig_2.0.3

**Comment by davidgohel** [comment]
Thanks, this has been fixed in the GitHub version, it should go on CRAN on Monday. The version you have and the cran version have this bug.

**Comment by svencisco** [comment]
Thanks so much for fixing this so quickly David! I'm seeing the same issue with body_add_img. Can I assume it's the same root cause and will be fixed as well?

**Comment by davidgohel** [comment]
@svencisco you can install the gh version to test it and make sure it works for you

**Comment by svencisco** [comment]
@davidgohel I can confirm the github version fixes the triple image issue with body_add_img. Thanks!

**Comment by davidgohel** [comment]
fixed in v 0.3.14, now on cran
